app/models/game.rb

```diff
@@ -1,13 +1,17 @@
 class Game < ApplicationRecord
 
+  scope :available, -> { where(black_player_id: nil) }
+
   after_create :set_pieces_on_board
   after_create :set_default_turn
 
-  belongs_to :white_player, class_name: "User", foreign_key:"white_player_id"
-  belongs_to :black_player, class_name: "User", foreign_key:"black_player_id"
+  belongs_to :white_player, class_name: "User", foreign_key:"white_player_id", required: false
+  belongs_to :black_player, class_name: "User", foreign_key:"black_player_id", required: false
```

Comment:
I don't really know why, but I was unable to create any games that actually saved to the database without this command....Jeff was also having the same issue